SpeedyWeather/Project.toml

```
@@ -0,0 +1,3 @@
+[deps]
+ParticleDA = "61cd1fb4-f4c4-4bc8-80c6-ea5639a6ca2e"
+SpeedyWeather = "9e226e20-d153-4fed-8a5b-493def4f21a9"
```

I always recommend using compat bounds
```suggestion
SpeedyWeather = "9e226e20-d153-4fed-8a5b-493def4f21a9"

[compat]
ParticleDA = "1.1"
SpeedyWeather = "0.12"
```